test/integration/flagging_two_step_verification_test.rb

```diff
@@ -0,0 +1,111 @@
+require "test_helper"
+
+class FlaggingTwoStepVerificationTest < ActionDispatch::IntegrationTest
+  include EmailHelpers
+  include ActiveJob::TestHelper
+
+  def sign_in_as_and_edit_user(sign_in_as, user_to_edit)
+    visit root_path
+    signin_with(sign_in_as)
+    visit edit_user_path(user_to_edit)
+  end
+
+  def assert_admin_can_send_2fa_email(admin, user)
+    sign_in_as_and_edit_user(admin, user)
+
+    perform_enqueued_jobs do
+      check "Ask user to set up 2-step verification"
+      click_button "Update User"
+
+      assert last_email
+      assert_equal "Make your Signon account more secure", last_email.subject
+    end
+
+    assert user.reload.require_2sv
+  end
+
+  def assert_admin_can_remove_2sv_requirement_without_notifying_user(admin, user)
+    sign_in_as_and_edit_user(admin, user)
+
+    perform_enqueued_jobs do
+      uncheck "Ask user to set up 2-step verification"
+      click_button "Update User"
+
+      assert_not last_email
+    end
+
+    assert_not user.reload.require_2sv
+  end
+
+  context "updating a user" do
```

Comment:
I don't think we need this context, as it seems to be for all of the tests.

In general, at least [in RSpec parlance](https://rspec.rubystyle.guide/#context-cases), a context should always start with a "when or a with" (our RSpec validator enforces that).